iOS/BrightcovePlayer/Playable.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Playable.swift
+//  BrightcovePlayer
+//
+//  Created by Roman Karpievich on 3/14/19.
+//  Copyright © 2019 Alex Faizullov. All rights reserved.
+//
+
+import Foundation
+import ZappPlugins
+
+class Playable: NSObject, ZPPlayable {
```

Comment:
We already have a class named `APURLPlayable` inside `ApplicasterSDK`, please compare the two and tell why do you need a new class. :)

Reply:
I removed `ApplicasterSDK` from dependencies so I can't use `APURLPlayable`.

Reply:
So you can bring it back and use the relevant model from there. 
Ping me on slack if you need more info. :)